tflearn/helpers/trainer.py

```diff
@@ -87,8 +87,11 @@ def __init__(self, train_ops, graph=None, clip_gradients=5.0,
             self.validate_trainop_names()
 
             self.global_loss = None
-            self.global_step = tf.Variable(0., name='Global_Step',
-                                           trainable=False)
+            if global_step:
```

Comment:
Here you need to set:
```
if global_step is None:
```
because 'if `Tensor`' raises an exception in TensorFlow.